src/lib/assets/icons/IconMinus.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+type IconMinusProps = {
+  title?: string
+}
+
+const IconMinus = (props: React.SVGProps<SVGSVGElement> & IconMinusProps) => (
+  <svg
+    {...props}
+    className={`Icon-Minus ${props.className ?? ''}`}
+    width="15"
+    height="2"
+    viewBox="0 0 15 2"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    {props.title && <title>{props.title}</title>}
+    <path d="M14.3887 2H8.38867H6.38867H0.388672V0H6.38867H8.38867H14.3887V2Z" />
+  </svg>
+)
+
+export default IconMinus
```

Comment:
I am exploring a different idea to generalize the icon component this way: https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/main/src/lib/containers/IconSvg.tsx

But I haven't gotten far enough to refactor all the icons yet. I think for now it's okay to create individual component for each icon.